### PR TITLE
Update to SDK with extended module account permissions

### PR DESCRIPTION
**Pull request by colin-axner**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry: `clog add [section] [stanza] [message]`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by tac0turtle — APPROVED** [review]
LGTM ⚡️

**Review by fedekunze — APPROVED** [review]
LGTM although the sims are not running @alessio . @colin-axner we should update the order of the init genesis too here to match the `simapp`'s one, basically moving `supply` between `crisis` and `mint`

File: go.sum [hunk 1]
```diff
@@ -39,6 +39,9 @@ github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e/go.mod h1:F5haX7
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
 github.com/cosmos/cosmos-sdk v0.28.2-0.20190705123402-2fdbf631e763 h1:25yfmbpVGN0+5yijzd4XhAMx0tJ/y5Jrec0gRqczjuA=
 github.com/cosmos/cosmos-sdk v0.28.2-0.20190705123402-2fdbf631e763/go.mod h1:ngmaRot5kvm4Op0NBgOgbOTWJOKw9cM2qAXv0DUs94Q=
+github.com/cosmos/cosmos-sdk v0.28.2-0.20190706194206-1c9a18834645 h1:dhMIDxDb3gHZLUVmMLPbULRE9jR1JbLsAd8Vf/c5ZGA=
+github.com/cosmos/cosmos-sdk v0.28.2-0.20190706194206-1c9a18834645/go.mod h1:ngmaRot5kvm4Op0NBgOgbOTWJOKw9cM2qAXv0DUs94Q=
+github.com/cosmos/cosmos-sdk v0.35.0 h1:EPeie1aKHwnXtTzKggvabG7aAPN+DDmju2xquvjFwao=
```
Comment (alessio) [inline go.sum]: please run `go mod tidy`

**Review by jackzampolin — APPROVED** [review]

**Comment by rigelrozanski** [comment]
Why was dis closed? (just curious)

**Comment by colin-axner** [comment]
#70 got merged instead which had the same changes, but used a different commit for go.mod. This pr probably should have been merged instead, as the other pr was meant to be used for testing with the supply genesis pr that was a WIP at the time.

On a different note, I wanted to open #70 as draft originally, but this repo wouldn't let me do so. Is this something we can enable or is there a reason for not allowing this?

**Comment by rigelrozanski** [comment]
no clue on the drafts... yeah maybe it has to be enabled per repo? (CC @jackzampolin can you do this?)

**Comment by jackzampolin** [comment]
Looks like you can do that here @rigelrozanski, no admin needed.

**Pull request closed without merging**